### PR TITLE
fix #225 add test for utils, fix utils

### LINKED ISSUE #225: Write unit tests for utils.py

### PR DESCRIPTION
**Pull request by fritjof-b**
Tests for utils, except `EarlyStopping`. Add `.item()` call to some methods in utils.

### CODE REVIEW

**Comment by fritjof-b** [comment]
This is failing because:

```
FileNotFoundError: [Errno 2] No such file or directory: 'projects/example_CMS/compressed_output/compressed.npz'
```

File: baler/modules/utils.py [hunk 1]
```diff
@@ -52,15 +52,15 @@ def mse_loss_emd_l1(model_children, true_data, reconstructed_data, reg_param, va
...
+    l1_loss = 0.0
     values = true_data
     if not validate:
         for i in range(len(model_children)):
             values = model_children[i](values)
             l1_loss += torch.mean(torch.abs(values))
 
         loss = emd_loss + mse_loss + reg_param * l1_loss
-        return loss, emd_loss, l1_loss
+        return loss, emd_loss, l1_loss.item()
```
Comment (fritjof-b) [inline baler/modules/utils.py:63]:
```suggestion
        return loss, emd_loss, l1_loss
```

File: baler/modules/utils.py [hunk 2]
```diff
@@ -100,9 +100,9 @@ def mse_loss_l1(model_children, true_data, reconstructed_data, reg_param, valida
             l1_loss += torch.mean(torch.abs(values))
 
         loss = mse_loss + reg_param * l1_loss
-        return loss, mse_loss, l1_loss
+        return loss, mse_loss.item(), l1_loss.item()
```
Comment (fritjof-b) [inline baler/modules/utils.py:103]:
```suggestion
        return loss, mse_loss, l1_loss
```

File: baler/modules/utils.py [hunk 3]
```diff
@@ -100,9 +100,9 @@ def mse_loss_l1(model_children, true_data, reconstructed_data, reg_param, valida
...
     else:
-        return mse_loss, 0, 0
+        return mse_loss.item(), 0, 0
```
Comment (fritjof-b) [inline baler/modules/utils.py:105]:
```suggestion
        return mse_loss, 0, 0
```

**Review by exook — COMMENTED** [review]
That is a lot of tests, thank you for this substantial contribution!

**Review by exook — APPROVED** [review]

**Pull request merged**